### DISCUSSION TITLE
/hyndsight/hardforecasts/

### DISCUSSION THREAD

**Discussion by robjhyndman** [robjhyndman/robjhyndman.com#123] [Announcements]
Why are some things easier to forecast than others? 
 https://robjhyndman.com/hyndsight/hardforecasts/

**Comment by robjhyndman** [comment]
```header
Originally posted by BuggyFunBunny on 2012-09-18 13:06:06
````
You should start with the Wikepedia article on efficient markets.
Forecasts, of foreign exchange in particular, are generally proprietary
prognostications for the privy privileged. IOW, sequestered information,
which is the antithesis of the EM hypothesis.

In a nutshell, what makes forecasting difficult is what kind of process
you\'re attempting to forecast. Weather does well since it is purely
mechanistic (although with so many parameters that supercomputers are
needed to track it all), while predicting any part of economic activity
is fraught with &quot;Danger, Will Robinson!!&quot;

The Great Recession (and one may be brewing in Oz?); the forecasters
could have gotten it right if they\'d been better economists, who
understood the fundamental ratios in the housing market ( median price /
median income, is one key). And knew that Greenspan had crashed interest
rates in 2002. And that China, due to its macro policies, generated
outlier amounts of moolah. And that the wealthy (of all locations)
demanded better &quot;risk free returns&quot; than US debt was then offering
(and still does). And that US mortgage companies (not so much banks)
were free to make any qualifying rules they felt like. And that
Banksters (Spain\'s problem, too) were more than happy to play sausage
maker with those loans. The list goes on.

IOW, policy (making it gives one a real leg up, knowing it before most
others, a smaller one) trumps data every time. And, no, I\'m not in the
Black Swan camp. There are no co-incidences or Black Swans; just
insufficient data.

**Comment by robjhyndman** [comment]
```header
Originally posted by davidreilly7 on 2017-12-20 19:32:42
````
Hi Rob, you mention the stock market here, and I recall your mentioning
in a YouTube video that for the stock market, naive is as good as it
gets.

So extending this to the broader use of Technical Analysis (especially
in volatile markets), is it your opinion that the whole field is largely
nonsensical?

**Reply by robjhyndman** [reply]
```header
Originally posted on 2017-12-20 21:45:29
````
A lot of it is nonsense, IMO. But there are some subtleties. See Roy
Batchelor\'s work in this area:
[https://www.cass.city.ac.uk\...](https://www.cass.city.ac.uk/faculties-and-research/experts/roy-batchelor &quot;https://www.cass.city.ac.uk/faculties-and-research/experts/roy-batchelor&quot;)

**Reply by robjhyndman** [reply]
```header
Originally posted by davidreilly7 on 2017-12-20 21:46:59
````
Thank you very much!

**Reply by robjhyndman** [reply]
```header
Originally posted by tli on 2018-06-24 21:27:44
````
Hi Rob,

I\'ve been going through your book Forecasting: Principles and Practice.
It\'s been very enlightening. However, I\'m curious regarding your
thoughts on stock price prediction.

Are there any papers you\'ve written specifically on this or other works
you recommend? You seem to think a lot of it is nonsense and most of the
links I\'ve found of people posting their works seems to make that case.
However, there\'s no doubt some major hedge funds like Renaissance
Technologies and 2Sigma (and several others) have managed to be
incredibly successful with quantitative investing.

I\'m wondering if you have any insights into what provides them their
edge? Especially given the buzzwords of Machine Learning floating
around. From my own reading and research, it\'s definitely what people
call statistical arbitrage. They are betting over a large number of
stocks to go up and down on a neutral dollar basis. Given all the noise
in stocks, the &quot;models&quot; are not very &quot;accurate&quot; but they do allow
them some edge (i.e 54%, 55% successful prediction is my guess).
Therefore they place bets on hundreds and/ore thousands of stocks (long
and short). If you look at their holdings, you can see weightings of
their portfolio for each stock is very similar and quite small (i.e. 1%
or lower for each stock).

My main question is, what type of models do you think they and using and
what type of timeframe?

They clearly are using a lot of features that on their own have very
faint signal. I\'ve also read that many times these features often lose
their signal but they leave them in the model just in case they come
back. In terms of features, from people I\'ve spoken to it\'s a mix of
technical, fundamental and &quot;other&quot; features (i.e. sentiment from news,
voice analysis from earnings calls, surveys, etc.).

I come from a fundamental investment perspective but I have tried to do
some research on what quantitive hedge funds have been doing. I\'d be
curious of any thoughts from what I\'ve shared.

In other words, if you have had many features, in a very noisy market,
where would you start?

Best,

Alex

**Reply by robjhyndman** [reply]
```header
Originally posted on 2018-06-24 22:00:12
````
You would have to ask someone who knows more about finance than me.

**Reply by robjhyndman** [reply]
```header
Originally posted by tli on 2018-06-24 22:32:21
````
I see. Thank you. In your book, you go over dynamic regression models.
Is there any theoretical reason why this approach would not work? It
seems directly applicable.

**Reply by robjhyndman** [reply]
```header
Originally posted on 2018-06-24 22:40:07
````
You need good predictors that you can forecast. Often the predictors are
harder to forecast than the original series.

**Reply by robjhyndman** [reply]
```header
Originally posted by tli on 2018-06-24 23:07:54
````
What do you mean by predictors that you can forecast? For example if
I\'m forecasting stock price as my y variable (or return or log of
prices to make them stationary), my possible features could be the
following:

1\) profit margin\
2) earnings\
3) sales growth\
4) price return t-1, t-2\
5) slope t-1, t-2\
6) moving averages\
7) technical analysis like bollinger bands\
8) sentiment analysis

I presume these would be historical data or else I would be forecasting
(sales estimates, earnings estimates, like a fundamental analyst - which
I was). But wouldn\'t that just bring in enormous bias.

Are you saying just using historical data isn\'t enough? I guess that
makes sense and why I\'ve heard them using sentiment analysis. But never
thought they were actually forecasting earnings.

**Reply by robjhyndman** [reply]
```header
Originally posted on 2018-06-24 23:08:55
````
Read
[https://otexts.org/fpp2/for\...](https://otexts.org/fpp2/forecasting.html)
and
[https://otexts.org/fpp2/for\...](https://otexts.org/fpp2/forecasting-regression.html &quot;https://otexts.org/fpp2/forecasting-regression.html&quot;)
